cmd/uplink/cmd/root.go

```diff
@@ -96,21 +96,23 @@ func (c *UplinkFlags) GetProject(ctx context.Context) (*libuplink.Project, error
 	cfg.Volatile.MaxInlineSize = c.Client.MaxInlineSize
 	cfg.Volatile.MaxMemory = c.RS.MaxBufferMem
 
-	uplink, err := c.NewUplink(ctx, cfg)
+	uplk, err := c.NewUplink(ctx, cfg)
 	if err != nil {
 		return nil, err
 	}
 
-	opts := &libuplink.ProjectOptions{}
+	encryptionKey, err := c.Enc.LoadKey()
+	if err != nil {
+		return nil, err
+	}
 
-	encKey := new(storj.Key)
-	copy(encKey[:], c.Enc.Key)
-	opts.Volatile.EncryptionKey = encKey
+	opts := &libuplink.ProjectOptions{}
+	opts.Volatile.EncryptionKey = &encryptionKey
```

Comment:
just a heads up that this will need to happen this next sprint: https://github.com/storj/storj/pull/1854